### PR TITLE
README update

### PR DESCRIPTION
**Pull request by saiyam1814**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,9 +22,8 @@ Civo CLI is a tool to manage your [Civo.com](https://www.civo.com) account from
 - [Load Balancers](#load-balancers)
 - [Quota](#quota)
 - [Sizes](#sizes)
-- [Snapshots](#snapshots)
 - [SSH Keys](#ssh-keys)
-- [Templates](#templates)
+- [DiskImages](#templates)
```
Comment (zulh-civo) [inline README.md:26]: The anchor link should be `#disk-image` here

**Review by zulh-civo — CHANGES_REQUESTED** [review]
I've had a quick eyeball review. Left a comment. Looks good overall.

Moving forward, I hope you can create a separate source branch for every PR. Since you are using `saiyam1814:main` branch here again, you are carrying the [merged commits](https://github.com/civo/cli/pull/135) from that branch.

File: README.md (hunk 1 above)
Comment (zulh-civo) [reply README.md:26]: Got a request from @kunal-kushwaha to merge this as-is. He will do the fixes in another PR.

**Review by zulh-civo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kunal-kushwaha) [reply README.md:26]: Linking here for reference: https://github.com/civo/cli/pull/150

**Pull request merged**